features/breol_frontend/breol_frontend.file_default_displays.inc

```diff
@@ -10,6 +10,83 @@
 function breol_frontend_file_default_displays() {
   $export = array();
 
+  $file_display = new stdClass();
```

Comment:
Why do an pull request about coding standards require adding some new display?

Reply:
It doesn't! I don't know where this came from. Nice catch!